models/org.go

```diff
@@ -737,6 +737,7 @@ func (org *User) GetUserTeams(userID int64) ([]*Team, error) {
 type AccessibleReposEnvironment interface {
 	CountRepos() (int64, error)
 	RepoIDs(page, pageSize int) ([]int64, error)
```

Comment:
Could we use an option for `RepoIDs` but not a new function?

Reply:
* The code I added here isn't even used. I'm taking it out again.
* Do you feel we should address the (imho unrelated) `RepoIDs` issue in this PR, or can we open a separate issue/PR for it?